docs/reference/images.md

```diff
@@ -453,4 +453,45 @@ $ sudo ig image inspect advise_seccomp:latest --show-datasources --jsonpath='[0]
   "runtime.containerImageDigest",
   "runtime.containerStartedAt"
 ]
-```
\ No newline at end of file
+
+```
+
+#### `verify`
+
+Verify the given gadget image signature.
+
+Before verifying an image, make sure it is already present in the local store.  
+If not, pull it first:
+
+```bash
+$ sudo ig image pull IMAGE_NAME
+```
+
+```bash
+$ sudo ig image verify -h
+
+Verify the given gadget image signature, for more details see [the documentation related to verifying](verify-assets.mdx).
+
+Usage:
+  ig image verify [gadget] [flags]
+
+Flags:
+  -h, --help                              help for verify
+      --notation-certificates string      Certificates used to verify the gadgets with notation
+      --notation-policy-document string   Policy Document used to verify the gadgets with notation
+      --public-keys string                Public keys used to verify the gadgets with cosign (default "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEoDOC0gYSxZTopenGmX3ZFvQ1DSfh\nIr4EKRt5jC+mXaJ7c7J+oREskYMn/SfZdRHNSOjLTZUMDm60zpXGhkFecg==\n-----END PUBLIC KEY-----\n")
+
+Global Flags:
+      --auto-mount-filesystems   Automatically mount bpffs, debugfs and tracefs if they are not already mounted
+      --auto-wsl-workaround      Automatically find the host procfs when running in WSL2
+      --config string            config file to use
+      --pprof-addr string        Starts a pprof server for profiling at the given address (e.g., 'localhost:6060'), leave empty to disable (default).
+  -v, --verbose                  Print debug information
+
+```
+
+```bash
+$ sudo ig image verify ghcr.io/inspektor-gadget/gadget/trace_exec:v0.45.0
+ 
+Verifying image: trace_exec:v0.45.0
+Image verified successfully!
```

Comment:
```suggestion
Verifying image: trace_exec:v0.45.0
Image verified successfully!
```

Also, add 3 backquotes (`\``) to close the code block and ensure the file is finished with an empty line.